mleap-core/src/main/scala/ml/combust/mleap/core/feature/MathUnaryModel.scala

```diff
@@ -56,3 +61,17 @@ case class MathUnaryModel(operation: UnaryOperation) extends Model {
   override def outputSchema: StructType = StructType(
     "output" -> ScalarType.Double.nonNullable).get
 }
+
+object LogitHelper { 
+  val LOGIT_MIN = 0.000001
+  val LOGIT_MAX = 0.999999
+  
+  def logit(value: Double): Double = {
+    if (value < LOGIT_MIN || value > LOGIT_MAX) {
+      throw new RuntimeException(s"logit value $value out of bounds")
```

Comment:
I don't think this is necessary because `commons-math` already throws an `OutOfRangeException` on line 162 of [this file](https://commons.apache.org/proper/commons-math/javadocs/api-3.5/src-html/org/apache/commons/math3/analysis/function/Logit.html#line.162)

Reply:
removing this